imports/client/components/PuzzleAnswer.tsx

```diff
@@ -38,9 +38,9 @@ const PuzzleAnswer = React.memo(({
     // Use Intl.Segmenter (stage 3 proposal) if available to properly segment grapheme clusters
     // Typescript is unaware of it, so there are a few any casts...
     let graphemes:string[];
-    if (Intl !== undefined && (Intl as any).Segmenter !== undefined) {
```

Comment:
Are these just places where upstream's type annotations got better, obviating our any casts?

Reply:
I suspect that we upgraded Typescript since this code was written, and the later versions of TS came with native declarations for the newer proposal where the older versions did not.